api/libproject/serviceaccounts.rb

```diff
@@ -0,0 +1,37 @@
+require_relative "../../aou-utils/workbench"
+require_relative "../../aou-utils/utils/common"
+
+# Puts the AppEngine service account credentials in "sa-key.json" for use in command line
+# utilities; sets GOOGLE_APPLICATION_CREDENTIALS to point at it, so application default
+# credentials will use it.
+# For environments other than test, deletes the credentials after use.
+class ServiceAccountContext
```

Comment:
I like how you pulled out this functionality. It's much clearer how this fits in.